uenv/wannier90/develop/daint/environtments.yaml

```
@@ -0,0 +1,16 @@
+gcc-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  mpi:
+      spec: cray-mpich@8.1.29
+  unify: true
+  specs:
+  - cmake
+  - openblas threads=openmp
+  - wannier90+mpi 
+  variants:
+  - +mpi
+  views:
+    default:
+      link: roots
```

Link `roots` won't include the dependencies. I had a few problems in the past, so I would keep the default here.
```suggestion
      link: all
```